webpack/move_to_foreman/components/common/table/components/CollapseSubscriptionGroupButton.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from 'patternfly-react';
+
+const CollapseSubscriptionGroupButton = ({
+  collapsed, visible, onClick, ...props
+}) => {
+  const iconName = collapsed ? 'angle-down' : 'angle-right';
+  const iconStyle = { visibility: visible ? 'visible' : 'hidden' };
+
+  return (
+    <Icon
+      className="collapse-subscription-group-button"
+      name={iconName}
+      style={iconStyle}
```

Comment:
could you use classnames? instead of inline styling

Reply:
I think that in this unique case I prefer the inline style.
It is actually not styling, it's on/off for the visibility.
Does it make sense to create extra css file only to define on/off for visibility? Won't it make it harder to understand the code in this case?
You will need an extra step, to open the css file and understand the difference between on/off

Reply:
I agree, this is a good usage of inline styling within a component.  It's still really odd to me to be in favor of inline styling in general but I can see the benefit here.